Content.Shared/GameTicking/SharedGameTicker.cs

```diff
@@ -144,18 +144,36 @@ public TickerJobsAvailableEvent(Dictionary<NetEntity, string> stationNames, Dict
         }
     }
 
-    [Serializable, NetSerializable]
-    public sealed class RoundEndMessageEvent : EntityEventArgs
+    [Serializable, NetSerializable, DataDefinition]
+    public sealed partial class RoundEndMessageEvent : EntityEventArgs
     {
-        [Serializable, NetSerializable]
-        public struct RoundEndPlayerInfo
+        [Serializable, NetSerializable, DataDefinition]
+        public partial struct RoundEndPlayerInfo
         {
+            [DataField]
             public string PlayerOOCName;
+
+            [DataField]
             public string? PlayerICName;
+
+            [DataField]
+            public string PlayerGuid;
```

Comment:
This should be stored as a `Guid` however the serializer doesn't currently support it so one moment engine PR incoming.